### PR TITLE
new rule to detect MIT license in URL

### PR DESCRIPTION
**Pull request by rakeshbalusa**
Added a new rule to detect license in the URL `https://github.com/scottjehl/Respond/blob/master/LICENSE-MIT`

### CODE REVIEW

File: src/licensedcode/data/rules/mit_url_1.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+licenses:
+    - mit
+template: no
```
Comment (pombredanne) [inline src/licensedcode/data/rules/mit_url_1.yml]: template: no is not needed if this is not a template

File: tests/licensedcode/data/licenses/mit_url_1.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+licenses:
+    - mit
+template: no
```
Comment (pombredanne) [inline tests/licensedcode/data/licenses/mit_url_1.yml]: `template: no` is not needed if this is not a template

Comment (rakeshbalusa) [inline tests/licensedcode/data/licenses/mit_url_1.yml]: ok...removing that line :+1:

**Comment by pombredanne** [comment]
@rakeshbalusa do you know if this was for a specific issue? If so, please add a comment with the issue # sowe have a link.
Thanks!

**Comment by rakeshbalusa** [comment]
ok, I will . I forget to put issue number sometimes.

File: tests/licensedcode/data/licenses/mit_url_1.yml (hunk 2 above)
Comment (pombredanne) [inline tests/licensedcode/data/licenses/mit_url_1.yml]: The line is still there...

**Pull request merged**